applicationloadbalancer/v0.0.1/main.tf

```diff
@@ -0,0 +1,151 @@
+#Create an application load balancer. Can be either public or internal.
+resource "aws_lb" "loadbalancer" {
+  name                       = var.name
+  internal                   = var.load_balancer_internal
+  load_balancer_type         = "application"
+  security_groups            = [aws_security_group.allow_lb.id]
+  subnets                    = var.subnet_ids
+  drop_invalid_header_fields = true
+
+  enable_deletion_protection = var.enable_deletion_protection
+  idle_timeout               = 3600
+  access_logs {
+    bucket  = aws_s3_bucket.lb_logs.bucket
+    prefix  = "${var.name}-lb_logs"
+    enabled = true
```

Comment:
I'd like to see this be optional.

Reply:
This whole access_logs block of code? @yannickvr

Reply:
Anything that's got to do with access logs. Not all environments require access logs to be enabled, so if we disable that, all features need to be disabled as well.